docs/examples/FastEmbed_Multi_GPU.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.            cache_dir="models",
</pre><p>why do we need to define it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qdrant/fastembed/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>